book/3-collab-dev-platforms/collaborative_ml.md

```diff
@@ -0,0 +1,35 @@
+# What are the platforms and their respective considerations required for collaborative ML development?
+
+Contributor(s): Dylan Poh Guan Kiong
+
+With the increasing complexity of Machine Learning projects, highly likely a team will be formed to complete the task. Being able to collaborate efficiently may be more critical than ever before. 
+
+Fundamentally, there are two ways that teams collaborate. <br>
+*Luke Marsden: When you boil it down to basics, there are really two fundamental modes of collaborating between different people doing work: synchronous and asynchronous.*
+
+*Synchronous*: Communications are scheduled, real-time interactions by phone, video, or in-person, sometimes interrupting one another for urgent tasks or simply asking a question. In terms of development, the upstream have to complete their tasks before the downstream team can continue with their work (Dependencies issues)
+
+*Asynchronous*: Communication happens on your own time, with a different team or people concurrently working on a different task. An efficient coping mechanism is required to resolve merged conflict. 
+
+With this in mind, we seek to introduce some tools and practices for a team to get started in the hope that stumbling blocks will be reduced to the minimum. The practices introduce here assume development using python as the primary language.
+
+## Communication tools.
```

Comment:
```## Communication tools.```
Remove full stop in title to be consistent.

Reply:
Updated.